symphony/bdk/core/activity/help_command.py

```diff
@@ -0,0 +1,22 @@
+from symphony.bdk.core.activity.command import CommandContext, SlashCommandActivity
+from symphony.bdk.core.service.message.model import Message
+from symphony.bdk.core.symphony_bdk import SymphonyBdk
+
+
+class HelpCommand(SlashCommandActivity):
+    """The help command is a particular CommandActivity which returns the list of all commands available for the
+    specific bot
+    The Slash command is triggered if we receive a MESSAGESENT event with message text:
+      - "@{bot_display name} /help"
+    """
+
+    def __init__(self, bdk: SymphonyBdk):
+        super().__init__("/help", True, bdk.messages(), "List available commands")
```

Comment:
IMO putting the MessageService instance as callback is misleading as this was designed to be a async function accepting a CommandContext, which will then be called when the message matches the parameters (command name and mention bot). I'd prefer to have:

- either None as callback and keep the overriden on_activity method
- or create a specific method and put it as callback in super()._init__

Reply:
I will change it to pass None as callback and I will call directly the bdk object inside the on_activity function